docs/topics/assembly_developing-and-deploying-vertx-application.adoc

```diff
@@ -20,6 +20,12 @@ In addition to xref:example-rest-http-vertx[using an example], you can create ne
 //Developing an application
 include::proc_developing-runtime-application.adoc[leveloffset=+1]
 
+//Supported OS and JDK images
+include::ref_openjdk-images-for-red-hat-enterprise-linux.adoc[leveloffset=+1]
+
+//Selecting the image to build and deploy your application
+include::proc_selecting-the-jdk-builder-image-for-your-application.adoc[leveloffset=+2]
```

Comment:
If the `leveloffset=+2` is intentional, then this module and the previous one must be in a new assembly, both with `leveloffset=+1`. ("nested" modules are not allowed under FCC, all adjacent modules must be on the same level)

Reply:
Ok. Are the FCC "rules" published somewhere?

Reply:
https://redhat-documentation.github.io/modular-docs/

Reply:
@tradej I don't see that nested modules are forbidden. Perhaps that is more to do with FCC than modular docs.

Reply:
That seems to be an unfortunate omission then. It has been the idea from the beginning (assemblies being the only files allowed to include other files). I'll file a JIRA for an inclusion of such a note.

Reply:
Filed: redhat-documentation/modular-docs#99.